app/views/admin/accounting/transactions/show.html.slim

```diff
@@ -0,0 +1,18 @@
+div
```

Comment:
We should reuse the form for new transactions. The transaction form needs to be split out from the modal content.

I will make that change.